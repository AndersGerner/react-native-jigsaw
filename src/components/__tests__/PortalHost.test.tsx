import * as React from "react";
import { View } from "react-native";
import renderer from "react-test-renderer";
import PortalHost from "../Portal/PortalHost";

it("renders portal host", () => {
  const tree = renderer.create(<PortalHost children={<View />} />).toJSON();

  expect(tree).toMatchSnapshot();
});
