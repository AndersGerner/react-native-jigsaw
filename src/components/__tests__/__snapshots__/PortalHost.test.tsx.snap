// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders portal host 1`] = `
<View
  collapsable={false}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View />
</View>
`;
